docs/getting-started.md

```diff
@@ -68,9 +70,15 @@ ssh-ed25519 [long random string] <user>@<host>
 
 </details>
 
+### Generate SSH key on Windows
+
+Windows requires the OpenSSH client, which has been permanently integrated into the system since the October 2018 Windows 10 update. If this is not available (so you can't use the command `ssh-keygen`), simply follow the instructions at the following link: [OpenSSH](https://learn.microsoft.com/en-us/windows-server/administration/openssh/openssh_install_firstuse?tabs=gui#install-openssh-using-windows-settings)
+
+Open Powershell and execute the command `ssh-keygen`. When prompted, make sure to choose a strong passphrase and save the passphrase in your password manager. The key pair (public and private) will appear by default at C:\Users\<username>\.ssh\ (Note that the name of this directory begins with a dot and is therefore invisible by default) inside your user directory. When done, send the contents of the public key with the file name id_rsa.pub with contact details (name and email adress) to your project coordinator.
+
 ### Setting up your SSH configuration
 
-To provide the appropriate parameters for the connection, create a file at `~/.ssh/config` and copy the snippet below, replacing `$USERNAME` appropriately.
+To provide the appropriate parameters for the connection, create a file at `~/.ssh/config` (Windows: `\Users\<username>\.ssh\config`) and copy the snippet below, replacing `$USERNAME` appropriately.
```

Comment:
Please verify that the config file contents don't need any changes on Windows.

Reply:
The config file looks exactly the same under Windows